contracts/ownership/Whitelist.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.4.18;
+
+
+import "./Ownable.sol";
+
+
+/**
+ * @title Whitelist
+ * @dev The Whitelist contract has a whitelist of addresses, and provides basic authorization control
+ * functions, this simplifies the implementation of "user permissions".
```

Comment:
let's add an `@dev` prefix here for multiple-line dev comments

Reply:
Done.